phpBB/phpbb/plupload/plupload.php

```diff
@@ -266,7 +266,7 @@ public function generate_resize_string()
 		if ($this->config['img_max_height'] > 0 && $this->config['img_max_width'] > 0)
 		{
 			$resize = sprintf(
-				'resize: {width: %d, height: %d, quality: 100},',
+				'resize: {width: %d, height: %d, quality: 90},',
```

Comment:
I'd recommend going down as far as 85. Should be plenty enough to still have a good enough quality.